marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/getOpticPlan.sjs

```diff
@@ -99,12 +100,26 @@ const qryTxt = cts.query(fn.head(searchTxtResponse));
 
 const ctsQry = cts.andQuery([qry, qryTxt]);
 
+let orderDefinitions = [];
+if (sortOrder) {
+  // convert ArrayNode to Array with .toObject()
+  for (let sort of sortOrder.toObject()) {
+    const col =  op.col(sort.name);
```

Comment:
We need to replace the hyphen with an underscore for these column names.
We can use the `replaceHyphenWithUnderscore` function here.